development/README.md

```diff
@@ -55,39 +55,18 @@ You can inspect the requests in the `Network` tab of your browser's Developer To
 by filtering for `POST` requests to `/v1/batch`. The full url will be `http://localhost:9090/v1/batch`
 or `https://api.segment.io/v1/batch` respectively.
 
-## Sentry
+## Debugging Sentry
 
-### Debugging in Sentry
+1. Set `SENTRY_DSN_DEV` in `.metamaskrc` to a suitable Sentry URL.
+  - The example value specified in `.metamaskrc.dist` uses the `test-metamask` project in the MetaMask account.
+  - Alternatively, create a free Sentry account with a new organization and project.
+  - The DSN is specified in: `Settings > Projects > [Project Name] > Client Keys (DSN)`.
 
-To debug in a production Sentry environment:
+2. To display Sentry logs, include `DEBUG=metamask:sentry:*` in `.metamaskrc`.
 
-- If you have not already got a Sentry account, you can create a free account on [Sentry](https://sentry.io/)
-- Create a New Sentry Organization
-    - If you already have an existing Sentry account and workspace, open the sidebar drop down menu, then click `Switch organization` followed by `Create a new organization`
-- Create a New Project
-- Copy the `Public Key` and `Project ID` from the Client Keys section under your projects Settings
-    - Select `Settings` in the sidebar menu, then select `Projects` in the secondary menu. Click your project then select `Client Keys (DSN)` from the secondary menu. Click the `Configure` button on the `Client Keys` page and copy your `Project Id` and `Public Key`
--   Add/replace the `SENTRY_DSN` and `SENTRY_DSN_DEV` variables in `.metamaskrc`
-    ```
-    SENTRY_DSN_DEV=https://{SENTRY_PUBLIC_KEY}@sentry.io/{SENTRY_PROJECT_ID}
-    SENTRY_DSN=https://{SENTRY_PUBLIC_KEY}@sentry.io/{SENTRY_PROJECT_ID}
```

Comment:
question: is this valid to remove `SENTRY_DSN` in env set?

Reply:
Have added a clarification of when to use this instead.